lib/galaxy/dependencies/pipfiles/default/pinned-requirements.txt

```diff
@@ -105,6 +110,7 @@ rfc3986==1.1.0
 routes==2.4.1
 simplejson==3.13.2
 six==1.11.0
+social_auth_core==1.5.0
```

Comment:
The new way to add dependencies is to add only the ones you directly use (which I think is just `social_auth_core`) to `lib/galaxy/dependencies/pipfiles/default/Pipfile` (without specifying a version unless needed) and run `make update-dependencies` to update this file ([pipenv](https://github.com/pypa/pipenv) needs to be installed).

Reply:
I changed it. Now I'm getting the following error, any thoughts? 

```shell
Invalid requirement: 'pipfiles/default/pinned-requirements.txt'
It looks like a path. Does it exist ?
```

Reply:
When does that happen?
I'd check that `lib/galaxy/dependencies/pinned-requirements.txt` is a symlink to `pipfiles/default/pinned-requirements.txt` . You can recreate it with:
```shell
cd lib/galaxy/dependencies/
rm -f pinned-requirements.txt
ln -s pipfiles/default/pinned-requirements.txt
```